spec/requests/articles/articles_spec.rb

```diff
@@ -51,14 +58,12 @@
 
     context "when :username param is given but it belongs to nither user nor organization" do
       include_context "when user/organization articles exist"
-      before { get "/feed", params: { username: "unknown" } }
-
-      it("renders empty body") { expect(response.body).to be_empty }
+      it("renders empty body") { expect { get "/feed", params: { username: "unknown" } }.to raise_error(ActiveRecord::RecordNotFound) }
     end
 
     context "when format is invalid" do
       it "returns a 404 response" do
-        expect { get "/feed.zip" }.to raise_error(ActionController::RoutingError)
+        expect { get "/feed.zip" }.to raise_error(ActiveRecord::RecordNotFound)
```

Comment:
Hmm...shouldn't a request for an unknown format render an HTTP 415 (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415). I'm unclear if that's appropriate for requesting an unsupported return type, or just for unsupported incoming payload types.

Reply:
Yeah, I think this is the most technically correct solution. But since the prior situation was also not correct I think it can be outside the scope of the PR.